string input_msg
geometry_msgs/Pose start_point
geometry_msgs/Pose end_point
---
string output_msg
geometry_msgs/Pose[] sampled_trajectory